modules/user-guide/pages/using-vale-in-the-ide.adoc

```diff
@@ -24,13 +24,6 @@ Consider using Vale in your IDE when:
 
 . In  Visual Studio Code, install the link:https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server[Visual Studio Code extension for Vale using the Marketplace].
 
-. Configure the extension to use Vale CLI rather than Vale Server  in the extension settings (*Preferences > Extensions > Vale > Use CLI*).
```

Comment:
Probably a separate issues/doc task. Do we need to mention or explain the 3 optional configurable options in the VS Code extension here?